source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
+
+[Committer Name] as you know, the ASF Infrastructure has set up your
+committer account with the username `[apacheID]`
+
+[If your project uses SVN]
+
+Please follow the instructions to set up your SSH,
+svn password, svn configuration, email forwarding, etc.
+https://www.apache.org/dev/#committers
+
+[If your project automatically adds committers as PMC members]
+Please subscribe to the [PROJECT] Project Management
+Committee mailing list private@[PROJECT].apache.org.
+[/If PMC]
+
+[PROJECT] has various resources at:
+  https://svn.apache.org/repos/asf/[PROJECT]
+
+[/If SVN]
+
+[If your project uses GitHub]
+You will now be able to merge approved PRs on GitHub for this project.
+
+You can manage your account settings at https://id.apache.org/
+[/If GitHub]
+
+As an ASF committer, you now also have commit access to specific
+sections of the ASF Foundation repository, as follows:
+
+The general "committers" at:
+https://svn.apache.org/repos/private/committers
+
+The developer section of the website describes roles within the ASF and 
+provides other resources:
+  https://www.apache.org/foundation/how-it-works.html
+  https://www.apache.org/dev/
+
+The incubator also has some useful information for new committers
+in incubating projects:
+  https://incubator.apache.org/guides/committer.html
+  https://incubator.apache.org/guides/ppmc.html
+
+Just as before you became a committer, participation in any ASF community
+requires adherence to the ASF Code of Conduct:
+  https://www.apache.org/foundation/policies/conduct.html
+
+[PROJECT should insert its own guidelines here; if none are available,
+ the Apache Forrest guidelines are available as a template.]
+  https://forrest.apache.org/guidelines.html
+
+If you have any questions during this phase, then please
+see the following resources:
+
+Apache developer's pages: https://www.apache.org/dev/
+Incubator committer guide: https://incubator.apache.org/guides/committer.html
+
+Naturally, if you don't understand anything be sure to ask us on the 
+[PROJECT] dev mailing list. Documentation is maintained by volunteers 
+and hence can be out-of-date and incomplete - of course you can now
+help fix that.
+
+A PMC member will announce your election to the dev list, and we
+encourage you to introduce yourself there.
+
+
```

Comment:
```suggestion
[YOUR NAME] on behalf of the PMC.
```

Also makes it more personal and explains that this message is sent on behalf of the PMC, not the person who sends it.